### ISSUE TITLE
storage in storage_pqset  #1069

### ISSUE THREAD

**Issue by wolfbunke** [openego/data_processing#167]
Is this task already solved?
See Redmine #1069
and: https://github.com/openego/data_processing/blob/dev/dataprocessing/sql_snippets/ego_dp_powerflow_timeseries_generator.sql#L9

**Comment by lukasol** [comment]
I don't think we need to assign pqset to storage since these are part of the dispatch and / or being optimized. 
Am I right?
@ulfmueller

**Comment by ulfmueller** [comment]
yes. unless we would want to calculate a lpf() or perform comparisons between 'global market simulation'/renpassGIS and 'after_redispatch'/lopf results. 
So I would leave this task open but with a low priority for right now.

**Comment by wolfbunke** [comment]
ok, thank you
